backend/src/auth/models.py

```diff
@@ -0,0 +1,7 @@
+from fastapi_users.db import SQLAlchemyBaseUserTableUUID
+
+from src.database import Base
+
+
+class User(SQLAlchemyBaseUserTableUUID, Base):
```

Comment:
Is there somewhere a documentation describing fields defined in the `SQLAlchemyBaseUserTableUUID` model? I'm asking because right now the schema for the `User` model is defined in the migration file (which can be confusing).